src/main/java/com/telefonica/iot/cygnus/sinks/OrionPostgreSQLSink.java

```diff
@@ -141,7 +141,7 @@ public void configure(Context context) {
         postgresqlPassword = context.getString("postgresql_password", "unknown");
```

Comment:
Yes, you're right. Same issue found in `OrionMySQLSink`. 

Fixed both in a272748